### DISCUSSION TITLE
computation graph export to tf usable?

### DISCUSSION THREAD

**Discussion by nilsbecker** [owlbarn/owl#591] [General]
hi, 

a while ago some experimental work demonstrated file-based export of owl computation graphs ane import into tensorflow. has this work been developed further? is it usable for training neural networks generated with owl on a gpu with tensorflow 2.5?

**Comment by mseri** [comment]
Yes, this is @jzstark work. It is usable afaik, it is released as `owl-symbolic`: https://discuss.ocaml.org/t/announce-release-of-owl-symbolic-0-1-0/4930

**Reply by nilsbecker** [reply]
i found it; will this be updated to support owl 1.0?

**Reply by mseri** [reply]
I thought it would work out of the box with it. I don't know, @jzstark is in charge of both new owl releases and owl-symbolic.

I have pinged him here, he can tell you more
